src/Appwrite/Auth/Validator/MockNumber.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+namespace Appwrite\Auth\Validator;
+
+use Utopia\Validator;
+use Utopia\Validator\Text;
+
+/**
+ * MockNumber.
+ *
+ * Validates if a given object represents a valid phone and OTP pair
+ */
+class MockNumber extends Validator
```

Comment:
Is `MockNumber` a good name here?

For me it's not descriptive enough...
but it's a tricky one

Some ideas:

MockPhoneOTPList
MockSMSAuthConfiguration
OTPTestingConfiguration

Reply:
I agree, this validator is confusing, if we have flat parameters, we can avoid this altogether

Reply:
MockPhoneOTPList makes most sense to me